### PR TITLE
Data mocking

### LINKED ISSUE #10: See a history of products that have been delivered to me

### LINKED ISSUE #12: Set a personal description - UX
As a user,
When I visit my profile page
I should see...

- [ ] a button to 'Add Personal Description'

### LINKED ISSUE #18: Edit shipping address
A user should be able to edit their shipping address.

On the user's profile page,
you should see...
- [ ] a button to 'Edit Shipping Address'

### LINKED ISSUE #20: Set a personal description - Functionality
As a user,
When I visit my profile page and click on 'Add Personal Description'
- [ ] I'm prompted to add a personal description
and when I hit submit
- [ ] I see the personal description on my profile page

### LINKED ISSUE #29: Edit Email Adress - UX
A user should be able to edit their email address.

On the user's profile page,
you should see...
- [ ] a button to 'Edit Email Address'

### PR DESCRIPTION
**Pull request by nathanielmillard**
### Participating Group Members:
* Nathaniel, Stacy, Ashley, and KJ
 
### Code Highlights:
* Primarily mocks api data for ux display
* Creates actions and reducers for updating user info
* Mocks data for item history

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing. 
- [ ] Yes, and all are passing.
 
### Issues: 
* Addresses #
* Closes #10 
* Closes #12
* Closes #18 
* Closes #20
* Closes #29 
 
### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: code/web/src/modules/user/Profile.js [hunk 1]
```diff
@@ -42,32 +42,59 @@ const Profile = (props) => (
       </GridCell>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
         <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
-        <p style={{ marginBottom: '0.5em' }}>Personal description goes here</p>
+        <p style={{ marginBottom: '0.5em' }}>{props.user.details.description ? props.user.details.description : "You don't have a description yet..."}</p>
         {/*Description text should be dynamic from user details ie props.user.details.description */}
       </GridCell>
```
Comment (stacyp2006) [inline code/web/src/modules/user/Profile.js:47]: We need to make sure to do a sweep before 5 and remove any unnecessary comments.

File: code/web/src/ui/image/Tile.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ const Tile = (props) => {
           background-image:url('${ image }');
           background-size: 100% auto;
           box-shadow: ${ shadow ? shadow : 'none' };
+          opacity: ${opacity ? opacity : 1}
         }
```
Comment (stacyp2006) [inline code/web/src/ui/image/Tile.js:20]: Great job on this. I think this is a nice ui feature and I'm glad we were able to get it implemented.

**Review by stacyp2006 — APPROVED** [review]

File: code/web/src/modules/user/api/actions.js [hunk 3]
```diff
@@ -117,3 +116,51 @@ export function getGenders() {
...
+  //Maybe won't get it.
+}
+
+export function updateUserInfo(state) {
+  return dispatch => {
+    dispatch({
+      details: state,
+      type: 'UPDATE_USER'
+    })
+  }
```
Comment (aemcdonald) [inline code/web/src/modules/user/api/actions.js:162]: We should remove these comments before 5 as well per Stacy's suggestion.

**Review by aemcdonald — APPROVED** [review]
Looks great, we will need to update our action tests to account for the new code.

**Review by kathrynljackson — APPROVED** [review]

**Pull request merged**